client/thin-replica-client/include/client/thin-replica-client/grpc_connection.hpp

```diff
@@ -154,24 +168,29 @@ class TrsConnection {
   // A state snapshot stream will be open after
   // openStateSnapshotStream returns, if and only if openStateSnapshotStream returns
   // Result::kSuccess; a stream will not be open in the failure and timeout cases.
-  virtual Result openStateSnapshotStream(const vmware::concord::replicastatesnapshot::StreamSnapshotRequest& request);
+  virtual Result openStateSnapshotStream(const vmware::concord::replicastatesnapshot::StreamSnapshotRequest& request,
+                                         RequestId& request_id);
 
+  virtual void cancelStateSnapshotStream(RequestId request_id);
   virtual void cancelStateSnapshotStream();
-  virtual bool hasStateSnapshotStream();
+  virtual bool hasStateSnapshotStream(RequestId request_id);
 
   // Read key values from an existing open state snapshot stream. Note readStateSnapshot
   // will automatically close the open state snapshot stream in the event it has to time
   // out the read.
-  virtual Result readStateSnapshot(vmware::concord::replicastatesnapshot::KeyValuePair* key_value);
+  virtual Result readStateSnapshot(RequestId request_id,
+                                   vmware::concord::replicastatesnapshot::StreamSnapshotResponse* snapshot_response);
 
   // Helper to print/log connection details
-  friend std::ostream& operator<<(std::ostream& os, const TrsConnection& trsc) {
+  friend std::ostream& operator<<(std::ostream& os, const GrpcConnection& trsc) {
     return os << trsc.client_id_ << " (" << trsc.address_ << ")";
   }
 
  protected:
   // Helper function to deal with gRPC
-  void createStub();
+  void createTRCStub();
+  void createRSSStub();
```

Comment:
According to the coding standard, the names should be `createTrcStub` and `createRssStub`.

Reply:
thanks @robem, I will change this in next commit.